plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.properties

```diff
@@ -0,0 +1,14 @@
+cloudsdkdeploy=App Engine Project Deploy (via Cloud SDK)
+
+cloudsdkfacetlabel=Google Cloud SDK
+cloudsdkfacetdescription=Adds support for App Engine through the Google Cloud SDK
+
+pluginName=Google Cloud SDK WTP Tools
```

Comment:
This description seems more general than it should.  _Local development support for AppEngine-Standard for Eclipse WTP_?